ci-operator/step-registry/ipi/conf/alibabacloud/cloud-creds-provision/ipi-conf-alibabacloud-cloud-creds-provision-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: ipi-conf-alibabacloud-cloud-creds-provision
+  from: cli
+  commands: ipi-conf-alibabacloud-cloud-creds-provision-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    The IPI cloud-creds-provision configure step adds a authentications.config.openshift.io/cluster object
+    and secrets for each operator.
```

Comment:
nit: "...for each credentials request."  Generally, our cred-requests map to operators, with some subset of our cluster operators requesting a single cred-request for each platform.  But as far as this step is concerned, we're just iterating over CredentialsRequests, and where they come from is orthogonal.